scripts/app-config-replace.js

```diff
@@ -21,6 +21,7 @@ const options = {
       appConfig.apiHost = API_HOST;
```

Comment:
would have to be `appConfig.apiHost = API_HOST + '/alfresco/aos'` to actually work, but it's not used by any tests at the moment afaik